infra/scripts/node.toml

```diff
@@ -51,10 +51,10 @@ mkdir -p $CMT_DIR
 
 [tasks.seed-enable]
 script="""
-sed -i'bak' "s/seed_mode = false/seed_mode = true/" ${CMT_DIR}/config/config.toml
+sed -i'' -e "s|seed_mode = false|seed_mode = true|" ${CMT_DIR}/config/config.toml
 """
 
 [tasks.set-seeds]
 script="""
-sed -i 's/seeds = ""/seeds = "${BOOTSTRAPS}"/g' ${CMT_DIR}/config/config.toml
+sed -i'' -e 's|seeds = ""|seeds = "${BOOTSTRAPS}"|g' ${CMT_DIR}/config/config.toml
```

Comment:
What's the `-e` doing? (Sorry)

Reply:
It is used to add a sed script. The goal of this is to create a sed command that works on MacOS and GNU/Linux systems.